client/components/marked-lines/style.scss

```diff
@@ -0,0 +1,46 @@
+.marked-lines {
+	padding: 0;
+	background-color: #e6ecf0;
```

Comment:
Should we use variables for this (and the other) colors?

Reply:
yeah I would love to - it's just that I'm pretty ignorant of what we have available and what w should be doing, so in this case I just brute-forced it by using a color detector on the mockups.

@keoshi are you willing to make decisions on the colors here? @MichaelArestad?

Reply:
Reviewing and making some changes shortly. :)